#163114828 Create connection model

ebenezerdon opened the pull request:
#### What does this PR do?
This PR creates Model and migration files for the `follow` table

#### Description of Task to be completed?
- Create migrations file for the connection table
- Add models file for the `connection` table

#### How should this be manually tested?
To manually test this:
- Run `db:migrate`
- Sequelize should run migrations for the new `connection` table

#### What are the relevant pivotal tracker stories?
The relevant pivotal tracker story for this PR:
 - #163114828 https://www.pivotaltracker.com/n/projects/2229853/stories/163114828

ebenezerdon committed:
(api): create Follow Model and migration file

ebenezerdon committed:
Merge branch with develop

ebenezerdon committed:
Merge branch 'develop' of https://github.com/andela/pippin-ah-backend into feature/163114828/create-follow-model

oreoluwade commented:
Here's two observations:

1. Given that there is `userId` on the Follow table/model, I would imagine that there's a relationship between the User and the Follow models, no?
2. Based on the above, if true, will a user have many followers or one follower per user?

ebenezerdon commented:
Yes, a user will have many followers

ebenezerdon commented:
I'll try to create a relationship between the User and Follow models and update my PR

ebenezerdon committed:
Pull develop to current branch

ebenezerdon committed:
Merge branch 'develop' of https://github.com/andela/pippin-ah-backend into feature/163114828/create-follow-model

ebenezerdon committed:
Rename models/follow.js to models/connection.js

ebenezerdon committed:
Change follow to connection
Use follwerId and follwedId instead of previous for easier compreahension

ebenezerdon committed:
feature(api): Create Connection Model and migration files
- Add migration file for connection
- Add model file for connection
[Finishes #Finishes]

ebenezerdon committed:
Merge branch 'develop' of https://github.com/andela/pippin-ah-backend into feature/163114828/create-follow-model

ebenezerdon committed:
feature(api): Create Connection Model and migration files
- Add migration file for connection
- Add model file for connection
[Finishes #163114828]
